src/control/settings/Settings.h

```diff
@@ -204,6 +204,12 @@ class Settings
 	void setLastSavePath(Path p);
 	Path getLastSavePath();
 
+    /**
+     * The last open path
+     */
+    void setLastOpenPath(Path p);
+	Path getLastOpenPath();
```

Comment:
The indents are somewhat awkward. No deal breaker just something to look for in the future.

Reply:
Oh, looks like that was before I set up my editor configuration to match the coding conventions (I use spaces by default). I'll fix that shortly.

Reply:
The formatting is not everywhere correct, even I did a complete reformatting some time ago.
There are so many developers now, it's hard to keep them im sync, I think features are more important than correct formatted code.... ;-)